queue/src/fiskaltrust.Middleware.Localization.QueueIT/v2/Receipt/PointOfSaleReceipt0x0001.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using fiskaltrust.ifPOS.v1;
+using fiskaltrust.ifPOS.v1.it;
+using fiskaltrust.Middleware.Localization.QueueIT.Constants;
+using fiskaltrust.Middleware.Localization.QueueIT.Extensions;
+using fiskaltrust.storage.V0;
+
+namespace fiskaltrust.Middleware.Localization.QueueIT.v2.Receipt
+{
+    public class PointOfSaleReceipt0x0001 : IReceiptTypeProcessor
+    {
+        private readonly IITSSCD _itSSCD;
+        private readonly IJournalITRepository _journalITRepository;
+
+        public ITReceiptCases ReceiptCase => ITReceiptCases.PointOfSaleReceipt0x0001;
+
+        public PointOfSaleReceipt0x0001(IITSSCD itSSCD, IJournalITRepository journalITRepository)
+        {
+            _itSSCD = itSSCD;
+            _journalITRepository = journalITRepository;
+        }
+
+        public async Task<(ReceiptResponse receiptResponse, List<ftActionJournal> actionJournals)> ExecuteAsync(ftQueue queue, ftQueueIT queueIt, ReceiptRequest request, ReceiptResponse receiptResponse, ftQueueItem queueItem)
+        {
+            var result = await _itSSCD.ProcessReceiptAsync(new ProcessRequest
+            {
+                ReceiptRequest = request,
+                ReceiptResponse = receiptResponse,
+            });
+            var documentNumber = result.ReceiptResponse.GetSignaturItem(SignatureTypesIT.RTDocumentNumber);
+            var zNumber = result.ReceiptResponse.GetSignaturItem(SignatureTypesIT.RTZNumber);
+            receiptResponse.ftReceiptIdentification += $"{zNumber.Data.PadLeft(4, '0')}-{documentNumber.Data.PadLeft(4, '0')}";
```

Comment:
Also `{zNumber.Data.PadLeft(4, '0')}` is repeated a lot 🤔

Reply:
Yep :/ will add an helper method for that.